test/behaviour/reasoner/verification/BoundPattern.java

```diff
@@ -156,21 +159,38 @@ public Concludable concludable() {
     static class BoundConclusion {
         private final BoundConjunction conjunction;
         private final Rule.Conclusion unboundConclusion;
+        private final Map<Variable, Concept> bounds;
 
-        private BoundConclusion(BoundConjunction conjunction, Rule.Conclusion unboundConclusion) {
+        private BoundConclusion(BoundConjunction conjunction, Rule.Conclusion unboundConclusion, Map<Variable, Concept> bounds) {
```

Comment:
suspicous that we have both `BoundConjunction` (presumably the conjunction and bounds), then the `unboundConclusion` (which contains the simple conjunction within the conclusion), and `bounds`... we should be able to derive the `BoundConclusion` from the other two, right?